apps/examples/testcase/ta_tc/media/utc/utc_media_bufferoutputdatasource.cpp

```diff
@@ -0,0 +1,179 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <media/BufferOutputDataSource.h>
+#include "tc_common.h"
+#include <errno.h>
+
+using namespace std;
+using namespace media;
+using namespace media::stream;
+
+static unsigned int channels = 2;
+static unsigned int sampleRate = 16000;
+static audio_format_type_t pcmFormat = AUDIO_FORMAT_TYPE_S16_LE;
+
+static ssize_t callback(unsigned char* buf, size_t size)
+{
+	return size;
```

Comment:
different argument size_t and return type is ssize_t
Please check it. 
I think return type "-1" mean negative case.

Reply:
To test more in detail put some codes that write data to file and play it when it finished.